### PR TITLE
feat: Inline Database

### PR DESCRIPTION
**Pull request by philippe-pixieset**
**Reference:** https://github.com/mariosimao/notion-sdk-php/issues/119

**Add the possibility to inline a database.**

I updated the unit and integration tests, and they both ran successfully in my environment.
LMK if you want me to change or add anything!

### CODE REVIEW

File: src/Databases/Database.php [hunk 1]
```diff
@@ -56,6 +57,7 @@ private function __construct(
         public readonly array $properties,
         public readonly DatabaseParent $parent,
         public readonly string $url,
+        public readonly bool $inline,
```
Comment (mariosimao) [inline src/Databases/Database.php:60]: I would rename it to `$isInline` to keep consistency across the SDK.

**Review by mariosimao — CHANGES_REQUESTED** [review]

**Comment by mariosimao** [comment]
@philippe-pixieset thank you for submitting this PR! I left a small review. Once the changes are applied, I will merge it to the main branch and release a new version.

File: src/Databases/Database.php (hunk 1 above)
Comment (philippe-pixieset) [reply src/Databases/Database.php]: Fixed!

**Review by mariosimao — APPROVED** [review]

**Comment by mariosimao** [comment]
@philippe-pixieset thanks again for your contribution. Version 1.13.0 has been released with support for inline databases.

**Comment by philippe-pixieset** [comment]
Thank you so much! And thank you for this repo 🙏🏻

**Pull request merged**